src/targets/slack.js

```diff
@@ -81,7 +81,10 @@ function setSuiteBlock({ result, target, payload }) {
         payload.blocks.push(getSuiteSummary({ target, suite }));
       }
       if (target.inputs.include_failure_details) {
-        payload.blocks.push(getFailureDetails(suite));
+        // Only attach failure details block if there were failures
+        if (result.failed > 0 ) {
```

Comment:
I think it should be `suite` instead of `result`
```suggestion
        if (suite.failed > 0 ) {
```

Reply:
You are correct.  It was still failing with multiple suites if one suite did not have failures.  Fixed.